go/libkb/globals.go

```diff
@@ -238,6 +238,7 @@ func (g *GlobalContext) PushShutdownHook(sh ShutdownHook) {
 }
 
 func (g *GlobalContext) ConfigureConfig() error {
+	RemoteSettingsRepairman(g)
```

Comment:
I did consider that, but... what do we want to happen? Is there repair attempt failure scenario that we'd want to be catastrophic? It's all logged anyway.